dev-client/src/navigation/constants.ts

```diff
@@ -38,7 +38,6 @@ export type TabStackParamList = {
   };
   [TabRoutes.SETTINGS]: {
     projectId: string;
-    downloadLink: string;
```

Comment:
Suggestion: move the removal of the downloadLink stuff to another PR, that also includes removing the associated UI and strings

Reply:
That was done elsewhere. Rebasing caused some merged conflicts, so now it's been doubly removed.